configs/config.yml

```diff
@@ -0,0 +1,34 @@
+log:
+  level: debug
+  pretty: true
+
+poller:
+  enabled: true
+  interval: 3000
+  blocks_per_poll: 3
+  from_block: 20000000
+  until_block: 20000010
+
+commiter:
+  enabled: true
+  interval: 10000
+  batch_size: 100
+
+failure_recoverer:
+  enabled: true
+  interval: 3000
+  batch_size: 100
```

Comment:
In which case this would be disabled?

Reply:
If you want to run failure recoverer separately